sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/lang/psi/sqlTypeName.kt

```diff
@@ -50,7 +50,8 @@ private fun MySqlTypeName.type(): IntermediateType {
     jsonDataType != null -> IntermediateType(IntermediateType.SqliteType.TEXT)
     enumSetType != null -> IntermediateType(IntermediateType.SqliteType.TEXT)
     characterType != null -> IntermediateType(IntermediateType.SqliteType.TEXT)
-    else -> throw IllegalArgumentException("Unknown kotlin type for sql type $this")
+    bitDataType != null -> IntermediateType(IntermediateType.SqliteType.INTEGER, BOOLEAN)
+    else -> throw IllegalArgumentException("Unknown kotlin type for sql type ${this.text}")
```

Comment:
Should we change the `else` clause of `PostgreSqlTypeName.type()` and `HsqlTypeName.type()` to use `${this.text}` for consistency?

Reply:
yup - will do